extensions/amp-lightbox-gallery/0.1/service/lightbox-manager-impl.js

```diff
@@ -188,20 +206,24 @@ export class LightboxManager {
    */
   processLightboxCarousel_(carousel) {
     const lightboxGroupId = carousel.getAttribute('lightbox') ||
-    'carousel' + (carousel.getAttribute('id') || this.counter_++);
+      `carousel${carousel.getAttribute('id') || this.counter_++}`;
+
     this.getSlidesFromCarousel_(carousel).then(slides => {
       slides.forEach(slide => {
-        const shouldExcludeSlide = slide.hasAttribute('lightbox-exclude')
-            || (slide.hasAttribute('lightbox')
+        const shouldExcludeSlide =
```

Comment:
This boolean assignment is getting complex. Maybe split a subexpression into another named boolean?

```js
const hasExistingLightbox = (slide.hasAttribute('lightbox') && 
    slide.getAttribute('lightbox') !== lightboxGroupId);
const shouldExcludeSlide = (slide.hasAttribute('lightbox-exclude') || hasExistingLightbox);
```

Reply:
This assignment is exactly the same as before, just indented differently.